src/plugins/interfaces.ts

```diff
@@ -0,0 +1,11 @@
+import PluginScope from './PluginScope';
```

Comment:
`types.ts` could be a better name than `interfaces.ts`, as you may want to have mapped types here too